tsconfig.json

```diff
@@ -1,26 +0,0 @@
-{
-  "compilerOptions": {
-    "baseUrl": ".",
-    "target": "es5",
-    "lib": ["dom", "dom.iterable", "esnext"],
-    "allowJs": true,
-    "skipLibCheck": true,
-    "strict": false,
-    "forceConsistentCasingInFileNames": true,
-    "noEmit": true,
-    "esModuleInterop": true,
-    "module": "esnext",
-    "moduleResolution": "node",
-    "resolveJsonModule": true,
-    "isolatedModules": true,
-    "jsx": "preserve",
-    "paths": {
-      "@/components/*": ["components/*"],
-      "@/core/*": ["core/*"],
-      "@/logic/*": ["logic/*"],
-      "@/tools/*": ["tools/*"]
-    }
-  },
-  "exclude": ["node_modules"],
-  "include": ["next-env.d.ts", "@types/**/*", "**/*.js", "**/*.jsx", "**/*.ts", "**/*.tsx"]
-}
```

Comment:
Is there a replacement configuration or are we moving back to vanilla js?

Reply:
Very good question, I think we need to follow these docs:  
https://docusaurus.io/docs/typescript-support

Reply:
Yup sounds good. That should probably be in scope, as some things won't work without it I think.